test/e2e_kubeadm/networking_test.go

```diff
@@ -122,6 +121,39 @@ var _ = KubeadmDescribe("networking [setup-networking]", func() {
 			})
 		})
 	})
+	ginkgo.Context("dual-stack [Feature:IPv6DualStack]", func() {
+		ginkgo.Context("podSubnet", func() {
+			ginkgo.It("should be properly configured if specified in kubeadm-config", func() {
+				if !dualStack {
+					framework.Skipf("Skipping because cluster is not dual-stack")
+				}
+				if !podSubnetInKubeadmConfig {
+					framework.Skipf("Skipping because podSubnet was not specified in kubeadm-config")
+				}
+				cc := getClusterConfiguration(f.ClientSet)
+				if _, ok := cc["networking"]; ok {
+					netCC := cc["networking"].(map[interface{}]interface{})
+					if ps, ok := netCC["podSubnet"]; ok {
+						nodes, err := f.ClientSet.CoreV1().Nodes().List(metav1.ListOptions{})
+						framework.ExpectNoError(err)
```

Comment:
looks like we missed adding annotations here and for the previous test cases.

the common pattern should be:
```
framework.ExpectNoError(err, "annotation for this error goes here - e.g. "error listing nodes")
```

it's OK to add annotations for the above `framework.ExpectNoError(err)` in this file too ( "single-stack", "serviceSubnet"). sorry we've missed that.

Reply:
Done!